python-pytest.el

```diff
@@ -2,7 +2,7 @@
 
 ;; Author: wouter bolsterlee <wouter@bolsterl.ee>
 ;; Version: 1.0.0
-;; Package-Requires: ((emacs "24.4") (dash "2.12.0") (dash-functional "2.12.0") (magit-popup "2.12.0") (projectile "0.14.0") (s "1.12.0"))
+;; Package-Requires: ((emacs "24.4") (dash "2.12.0") (dash-functional "2.12.0") (transient "20200719") (projectile "0.14.0") (s "1.12.0"))
```

Comment:
seems like this is using melpa date-based versioning, while it's more common to usual actual version numbers here.

or am i missing something?

Reply:
Initially I went for `0.2.0`  - the latest available at melpa stable now. However, it's missing [this commit](https://github.com/magit/transient/commit/dd0c44cb2db4bed0239832c3dd0c82076824c080) that introduces package prefixed names. I went to `magit` to see [how it is done there](https://github.com/magit/magit/blob/798aff56457aef4d1896db8129ab56d08ae12066/lisp/git-commit.el#L14) and they use dated version. I'm happy update it to `0.3.0` when it is out, but would rather not to go to old names.

Reply:
makes sense, let's leave it as is